CommonMark.Portable45/CommonMark.Portable45.csproj

```diff
@@ -0,0 +1,175 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Alright. I think I got the `Import` trick, though it still feels a lot like black magic.